deployment/helm/node-feature-discovery/values.yaml

```
@@ -584,3 +584,47 @@ prometheus:
   enable: false
   scrapeInterval: 10s
   labels: {}
+
+spire:
+  enabled: false
+  global:
+    spire:
+      clusterName: "nfd"
+      trustDomain: "nfd.io"
```

Didn't catch my earlier, but I think it'd be best to s/nfd.io/nfd.k8s-sigs.io/ in this PR

```suggestion
      trustDomain: "nfd.k8s-sigs.io"
```

WDYT?

---

++ let's go for Markus recommendation

---

sure